juju/juju
# Use jammy by default on 3.0

Juju 3.0 will default to using jammy as the controller/unit preferred base-image.

## QA steps

`juju bootstrap localhost` => lxd container with jammy
`make microk8s-operator-update && juju bootstrap microk8s` => controller pod with a jammy based image
deploy an iaas and k8s charm that support jammy, should chose jammy

## Documentation changes

N/A

## Bug reference

N/A
